ansible/roles/schulcloud-server-provisioning/templates/schulconnex-group-provisioning-deployment.yml.j2

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: schulconnex-group-provisioning-deployment
+  namespace: {{ NAMESPACE }}
+{% if ANNOTATIONS is defined and ANNOTATIONS|bool %}
+  annotations:
+{% if RELOADER is defined and RELOADER|bool %}
+    reloader.stakater.com/auto: "true"
+{% endif %}
+{% endif %}
+  labels:
+    app: schulconnex-group-provisioning
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: schulconnex-group-provisioning
+    app.kubernetes.io/component: provisioning
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+spec:
+  replicas: {{ SCHULCONNEX_GROUP_PROVISIONING_REPLICAS }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      #maxUnavailable: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: schulconnex-group-provisioning
+  template:
+    metadata:
+      labels:
+        app: schulconnex-group-provisioning
+        app.kubernetes.io/part-of: schulcloud-verbund
+        app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        app.kubernetes.io/name: schulconnex-group-provisioning
+        app.kubernetes.io/component: provisioning
+        app.kubernetes.io/managed-by: ansible
+        git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+        git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: schulconnex-group-provisioning
+        image: {{ SCHULCLOUD_SERVER_IMAGE }}:{{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        imagePullPolicy: IfNotPresent
+        envFrom:
+        - configMapRef:
+            name: api-configmap
+        - secretRef:
+            name: api-secret
+        - secretRef:
+            name: api-files-secret
```

Comment:
Should have his own configmap and secret and not need api-configmap, api-secret and api-files-secret
So please add an configmap and secret for this deployment who has only what it need in it

Reply:
It seems like I don't have access to the variables of the [config.yml](https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/all/config.yml) in dof_app, if I use a custom configmap. That's probably why all other deployments still use the api-configmap. Do I have to put all of the variables into the instance_config_all.yml?

Reply:
The [config.yml](https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/all/config.yml) from dof_app_deploy is the wrong file.
As an example for this kind off configmaps https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/ansible/roles/schulcloud-server-core/templates/admin-api-server-configmap.yml.j2